### DISCUSSION TITLE
这是彩蛋？

### DISCUSSION THREAD

**Discussion by KawaiiSparkle** [share121/inter-knot#4626] [其他]
![qq_pic_merged_1725788347720](https://github.com/user-attachments/assets/7bcedfa1-6025-45cb-ad1d-f12bb8ddcfee)

**Comment by LMA936** [comment]
我趣，柚子厨（确实应该就是小春和芦花）
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by vinori233** [comment]
是的，柚子社经典前作人物背景回归，几乎每一作都有，b站上有合集你可以去看看૧(●´৺`●)૭

**Comment by ysdtal** [comment]
无处不在的柚子厨Ciallo～(∠・ω< )⌒☆​
